api_query/src/user.ts

```diff
@@ -19,11 +19,12 @@ export type UserQueryParameters = {
     name: string;
     region_id: number;
     sort: string[];
+    password: string;
```

Comment:
In dit object kan je `password` weglaten, deze wordt gebruikt om `GET` requests te filteren